packages/codemods/src/v8-v9/transforms/utils/index.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { NodePath } from '@babel/core';
+import { JSCodeshift, ASTPath, Collection, ImportDeclaration } from 'jscodeshift';
+
+// Replace node while preserving comments
+export function replaceWithComments(
+  j: JSCodeshift,
+  path: ASTPath<any>,
+  newNode: NodePath<any>['node']
+) {
+  if (path.node.comments) {
+    newNode.comments = path.node.comments;
+  }
+  j(path).replaceWith(newNode);
+}
+
+export function findRelevantImportDeclarations(
+  root: Collection<any>,
+  j: JSCodeshift,
+  importSource: string
+): Collection<ImportDeclaration> {
+  return root.find(j.ImportDeclaration, {
+    source: {
+      // ensure deep equal check
+      value: (val: string) => val === importSource
```

Comment:
Very important. The object literal form (ex: `{ source: { value: importSource } }`) appears to use `String.prototype.includes`, which has led to surprises with `react-dropdowns` and `react-dropdown.next`.